python/cudf/cudf/io/orc.py

```diff
@@ -171,7 +171,10 @@ def read_orc_statistics(
     files_statistics = []
     stripes_statistics = []
     for source in filepaths_or_buffers:
-        filepath_or_buffer, compression = ioutils.get_filepath_or_buffer(
+        (
+            filepath_or_buffer,
+            compression,
+        ) = ioutils.get_reader_filepath_or_buffer(
```

Comment:
Might be able to write this to fit on one line, if you use `path_or_buf` like the other functions calling this.

Reply:
Done